near-sdk-macros/src/lib.rs

```diff
@@ -68,12 +94,33 @@ pub fn near(attr: TokenStream, item: TokenStream) -> TokenStream {
     } else {
         quote! {::near_sdk}
     };
+
+    // Check for nested near macros by parsing the input and examining actual attributes
+    if has_nested_near_macros(item.clone()) {
+        return TokenStream::from(
+            syn::Error::new(
+                Span::call_site(),
+                "The `near` macro cannot be nested with other `near` or `near_bindgen` macros. Please comma-separate the attributes instead of nesting them",
```

Comment:
the text is a bit imprecise.

 `#[near]` or `#[near_bindgen]` attributes are not allowed to be nested inside of the outmost `#[near]` attribute.
 Only a single `#[near]` attribute is allowed.